frontend_components/encord_active_components/frontend/src/constants.ts

```diff
@@ -1,2 +1,2 @@
-export const apiUrl = import.meta.env.VITE_API_URL ?? "http://localhost:8000";
-export const env = import.meta.env.VITE_ENV || "local";
+export const apiUrl = "http://localhost:8000";
```

Comment:
can we make this similarly just a env variable? & extend the vite.config.ts logic to always set the value.

Reply:
Went with a slightly different solution but fixed the issue